client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/components/TabbedViewQuestions.tsx

```diff
@@ -49,29 +47,22 @@ const TabbedViewQuestions: FC<Props> = (props) => {
           nonLinear
         >
           {questionIds.map((id, index) => {
-            if (shouldRenderActiveStepper(index)) {
-              return (
-                <Step
-                  key={currentQuestionId}
-                  active={!autograded || index <= maxStep}
-                >
-                  <StepButton
-                    className="p-4"
-                    icon={
-                      <StepperButton
-                        questionId={id}
-                        questionIndex={index}
-                        stepIndex={stepIndex}
-                      />
-                    }
-                    onClick={() => setStepIndex(index)}
-                  />
-                </Step>
-              );
-            }
+            const shouldRender = shouldRenderActiveStepper(index);
```

Comment:
Nit:  
`const isDisabled = !shouldRenderActiveStepper(index);`  
or  
`const isEnabled = shouldRenderActiveStepper(index);`  
would be more meaninful than `shouldRender`. Otherwise, `shouldRenderActive` is more semantically correct.